smoke-test.devfile.yaml

```diff
@@ -0,0 +1,118 @@
+schemaVersion: 2.2.0
+metadata:
+  generateName: quarkus-api-example
+attributes:
+  controller.devfile.io/storage-type: ephemeral
+starterProjects:
+  - name: quarkus-api-example
+    git:
+      checkoutFrom:
+        revision: main
+      remotes:
+        origin: https://github.com/che-incubator/quarkus-api-example.git
```

Comment:
@vitaliy-guliy can we drop start projects and use identical devfile from main just with different name and limits?